PV260_Project/PresentationLayer/Program.cs

```diff
@@ -39,9 +39,11 @@ public class Program
                     services.AddTransient<IEmailUi, EmailUi>();
                     services.AddTransient<IDataSetUi, DataSetUi>();
                     services.AddTransient<IDataLoader, CsvFileLoader>();
+                    services.AddTransient<IDataDownloader, CsvFileDownloader>();
                     services.AddTransient<IDiffComputer, DiffComputer>();
                     Console.WriteLine(Directory.GetCurrentDirectory());
-                    services.AddDbContext<StockSystemDbContext>(options => options.UseSqlite("DataSource=..\\..\\..\\..\\DataLayer\\app.db"));
+                    // services.AddDbContext<StockSystemDbContext>(options => options.UseSqlite("DataSource=..\\..\\..\\..\\DataLayer\\app.db"));
+                    services.AddDbContext<StockSystemDbContext>(options => options.UseSqlite("DataSource=/home/rumec/Documents/c#/cv/PV260/PV260_Project/DataLayer/app.db"));
```

Comment:
I would keep the relative path uncommented, and comment out the absolute/specific path.

Reply:
Try if this works (or something similar) on your OS. `Path.Combine(Directory.GetParent(Environment.CurrentDirectory).Parent.Parent.FullName, "..", "DataLayer", "app.db")`

Reply:
Sorry, I forgot to change it in this commit.

The `Path.Combine` approach did not work for some reason. I will add a NOTE informing UNIX system users about this. AIFAK Mr. Goth uses Windows, so I will keep it like this.